packages/elements/src/slider/__test__/slider.test.js

```diff
@@ -464,4 +476,24 @@ describe('slider/Slider', function () {
       expect(el.stepRange).to.equal(5.5);
     });
   });
+
+  describe('Marker', function () {
+    it('Set position of marker correctly when new marker added', async function () {
+      const marker = document.createElement('ef-slider-marker');
+      marker.value = '20';
+      el.appendChild(marker);
+      await elementUpdated(el);
+      expect(marker.style.left).to.equal('20%');
+    });
+    it('Set position of marker correctly with min and max', async function () {
+      const el = await fixture(`
+      <ef-slider min="0" max="60">
+        <ef-slider-marker value="13">13</ef-slider-marker>
+        <ef-slider-marker value="29">29</ef-slider-marker>
+      </ef-slider>`);
+      const marker = el.querySelectorAll('ef-slider-marker');
+      expect(parseFloat(marker[0].style.left)).to.greaterThan(13);
+      expect(parseFloat(marker[1].style.left)).to.greaterThan(29);
```

Comment:
Let's check with `equal` instead of `greaterThan` as the position could be off by a large margin yet the test still passes.
```suggestion
        <ef-slider-marker value="15">15</ef-slider-marker>
        <ef-slider-marker value="30">30</ef-slider-marker>
      </ef-slider>`);
      const marker = el.querySelectorAll('ef-slider-marker');
      expect(marker[0].style.left).to.equal('25%');
      expect(marker[1].style.left).to.equal('50%');
```